### ISSUE TITLE
Run accessibility tests on EKS environment

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#3967]
## Description

This is a follow-on from #3289. Cypress accessibility tests are not currently run on BRD environments because of OS-level dependency issues. After migrating to EKS, we should ensure that accessibility tests are run on a production-like environment.

## Acceptance Criteria
- [ ] Cypress accessibility tests are run on a production-like environment and can fail builds

**Comment by acrollet** [comment]
@ElijahLynn I couldn't find an open issue for updating the BRD infrastructure to block this issue on.

**Comment by ElijahLynn** [comment]
After thinking about it, I am also kinda hesitant to install a bunch of Xorg stuff on the BRD servers. I wonder if we should just leave accessibility to CI only?

**Comment by acrollet** [comment]
Alternatively if BRD is docker-ized, perhaps we could install the deps and run the tests within a container that could then be destroyed? We might want to discuss as part of #4109...

**Comment by ElijahLynn** [comment]
We can chat about that. We will likely need a way to spawn many containers to parallelize the test suite in general. My one concern is that if the containers differ then there is always "that one thing" that is different that causes for different results. And it would be nice to have a container flow from PROD to CI to LOCAL. I think with your ideas we can come up with something workable though.
